### ISSUE TITLE
Changing Weight Decay value showing worst performance

### ISSUE THREAD

**Issue by Addhi86** [vlfeat/matconvnet#1051]
Hello

I am working on the binary classification. I have created my network like:
Conv1, Relu1, Pool1 - Conv2, Relu2, Pool2 - Conv3, Relu3, Pool3 - Conv4, Relu4 - Conv5 Relu5 Dropot 0.5, FC, Dropout 0.5 - SoftmaxlossLayer

All conv layer is 3x3.

The default Weightdecay is 0.0005. And I am getting this result.
![image](https://user-images.githubusercontent.com/22164098/30151463-e60736da-93f2-11e7-8fe5-43da38714f54.png)

The same network is then used with Weightdecay 0.005

![image](https://user-images.githubusercontent.com/22164098/30151496-0b20939e-93f3-11e7-81a0-2d1646a248b4.png)

Anyone, Please help me to share why it is showing like that by changing the weight decay value?
